hadoop-project/pom.xml

```diff
@@ -1199,6 +1196,12 @@
         <artifactId>junit-platform-launcher</artifactId>
         <version>${junit.platform.version}</version>
         <scope>test</scope>
+        <exclusions>
```

Comment:
Why do we need to exclude transitive dependencies on commons-logging? These libraries use commons-logging as logging facade, exclude it may cause problems?